src/main/java/step3/domian/Revenue.java

```diff
@@ -0,0 +1,40 @@
+package step3.domian;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Revenue {
+
+    private int revenue;
```

Comment:
```suggestion
    private final int revenue;
```
final 키워드를 붙이면 어떨까요?